src/configuration.c

```diff
@@ -169,6 +322,12 @@ static struct argp_option const options[] = {
           0,
           "Log to DEST (stderr, syslog or journal), e.g. --log=journal",
           0 },
+        { "notify-flags",
+          MPTCPD_NOTIFY_FLAGS_KEY,
+          "BOOL",
+          0,
+          "FLags for address notify notification, e.g. --notify-flags=existing,skip_link_local,skip_loopback",
```

Comment:
Shorten to `"Address notification flags, e.g. --notify-flags=existing,skip_link_local,skip_loopback"`?

Reply:
Agreed on the shorten form.

'BOOL' is left-over from a previous internal version 'FLAGS' should be more accurate